ls1intum/Artemis
# [Exam] randomize order for quizzes

### Checklist

### Motivation and Context


Quiz questions and quiz answer options are always presented in the same order in an exam despite being able to configure randomization when creating a quiz.

### Description

Quiz questions and quiz answer options are now presented in random order if the option is chosen.

### Steps for Testing


1. Log in to Artemis
2. Create an exam with multiple quiz exercises (try different combinations of the randomization)
3. Participate in the exam
4. Check that it is presented in random order for the quiz exercises / quiz answer options you enabled the option.
(Hint: quizzes are also randomized when refreshing)
